mediapipe/tasks/ios/vision/core/sources/MPPVisionTaskRunner.mm

```diff
@@ -70,4 +76,91 @@ - (nullable instancetype)initWithCalculatorGraphConfig:(CalculatorGraphConfig)gr
   return self;
 }
 
+- (std::optional<NormalizedRect>)normalizedRectFromRegionOfInterest:(CGRect)roi
+                                                   imageOrientation:
+                                                       (UIImageOrientation)imageOrientation
+                                                         roiAllowed:(BOOL)roiAllowed
+                                                              error:(NSError **)error {
+  if (CGRectEqualToRect(roi, CGRectZero) && !roiAllowed) {
```

Comment:
I think it's deprecated only in Swift since `Equatable` is exclusively a swift protocol.
When I use == in ObjC, I get an error saying: invalid operands to binary expression ('CGRect' and 'CGRect'). I think the docs are inconsistent.